apps/src/code-studio/components/progress/UnitOverviewHeader.jsx

```diff
@@ -165,7 +156,6 @@ class UnitOverviewHeader extends Component {
             notice={i18n.wrongCourseVersionWarningNotice()}
             details={versionWarningDetails}
             dismissible={true}
-            width={SCRIPT_OVERVIEW_WIDTH}
```

Comment:
Fixes announcements being wider than other page content and makes them width responsive.